Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/dialogs/MotionAlertDialog.java

```diff
@@ -49,7 +53,11 @@ public boolean onKeyEvent(int keyCode, KeyEvent event)
       case KeyEvent.ACTION_UP:
         if (!ControllerMappingHelper.shouldKeyBeIgnored(event.getDevice(), keyCode))
         {
-          saveKeyInput(event);
+          // Special case for Rumble
+          if (setting.getKey().contains(SettingsFile.KEY_EMU_RUMBLE))
```

Comment:
I am not sure if I get this right, but it sounds we are hacking the MotionAlertDialog + InputBindingSetting for a case that is not really a key binding. Isn't it more cleaner to create a new Setting type that for this specific case where we don't actually need to have a binding settings for the emulated rumble. Also this KEY_EMU_RUMBLE isn't actually an INI key that is needed to enable the rumble, is it?

Reply:
It now uses the value thats in the INI since EmulationActivity now loads the settings when it is created.